src/igvfd/schemas/award.json

```diff
@@ -128,7 +84,7 @@
             "type": "string",
```

Comment:
I am not sure i recall all the ways that viewing group affects permissions.  Could the description be more clear as to the use of this property?